src/sourcemaps/utils/other-wizards.ts

```diff
@@ -0,0 +1,125 @@
+// @ts-ignore - clack is ESM and TS complains about that. It works though
+import clack from '@clack/prompts';
+import chalk from 'chalk';
+import { runNextjsWizard } from '../../nextjs/nextjs-wizard';
+import { runSvelteKitWizard } from '../../sveltekit/sveltekit-wizard';
+
+import { abortIfCancelled, getPackageDotJson } from '../../utils/clack-utils';
+import { findPackageFromList, hasPackageInstalled } from '../../utils/package';
+
+import * as Sentry from '@sentry/node';
+import { abort } from 'process';
+import { WizardOptions } from '../../utils/types';
+
+type WizardFunction = (options: WizardOptions) => Promise<void>;
+
+type FrameworkInfo = {
+  frameworkName: string;
+  frameworkSlug: string;
+  frameworkPackage: string;
+  sourcemapsDocsLink: string;
+  wizard: WizardFunction;
+};
+
+const sdkMap: Record<string, FrameworkInfo> = {
+  '@sentry/nextjs': {
+    frameworkName: 'Next.js',
+    frameworkSlug: 'nextjs',
+    frameworkPackage: 'next',
+    sourcemapsDocsLink:
+      'https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/#configure-source-maps',
+    wizard: runNextjsWizard,
+  },
+  '@sentry/sveltekit': {
+    frameworkName: 'SvelteKit',
+    frameworkSlug: 'sveltekit',
+    frameworkPackage: '@sveltejs/kit',
+    sourcemapsDocsLink:
+      'https://docs.sentry.io/platforms/javascript/guides/sveltekit/manual-setup/#configure-source-maps-upload',
+    wizard: runSvelteKitWizard,
+  },
+};
+
+export async function redirectToMoreSuitableWizard(): Promise<
```

Comment:
From what I can see this function does two things:

- Check if a framework with better wizard is installed
- Redirects the flow if one was found

I would do either of two things:
- Rename the function to `checkIfMoreSuitableWizardExistsAndRedirect()`
- Extract the logic for checking whether next/sveltekit is installed out of this function.

The name of this function would lead me to believe that it redirects to another flow no matter what.